src/tests/system/tests/test_passkey.py

```diff
@@ -411,3 +411,41 @@ def test_passkey__su_same_key_for_multi_user(
             ioctl=f"{moduledatadir}/umockdev.ioctl",
             script=f"{testdatadir}/umockdev.script.{suffix}.{user}",
         )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.ticket(jira="SSSD-7011", gh=7066)
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__ssh_key_as_passkey_mapping_token(client: Client, provider: GenericADProvider):
+    """
+    :title: Check ssh key as a passkey mapping token with AD and Samba server.
+    :setup:
+        1. Add a users in AD and Samba server and add a ssh key a passkey mapping.
+        2. Setup SSSD client with FIDO, start SSSD service.
+    :steps:
+        1. Required error message in pam log.
+    :expectedresults:
+        1. Successfully get the expected message in pam log.
+    :customerscenario: False
+    """
+
+    ssh_key = (
+        "ssh-rsa\20AAAAB3NzaC1yc2EAAAADAQABAAABgQCa+l8uZ6Q5G58PVMe1na7NrOMTzo2wOZfFwo0fM3RbvfAdlz/ws"
+        "Gwln2+EXA19FiXu/nNj4EwYGP9hymKuYaXzpq40k0VbhEL1v/qzXQvuKZgNx42vxi7NITaaAXuYj8OZQsZTvv+xgkREZ"
+        "mhQ6YqEjTJ0JzpD9fj8Gf8Mgn8pdsb/ZODLMAwEKtQ2DaWqH5jCqzoGEJlRl+kRbnrHc+RQrmj7NnY1voEJNrmzCyJFH"
+        "5awZyBl/ZdbvpnwCKnVEleBFULrOIfJ9lc/QMmURCMa6RfW5CFrxdtjUwiIxfMiHe+zUY5T9L0Q6FWnlfNz/63Xdcrw1"
+        "Gc90OCZKcqf/4P9N5flGSGSfiO5fD8gCCJ0c3WhxSVMREDP3ibKDsz8yhw2OuyGcfRo4nnchxy9G7031m2t9rUXc12eS"
+        "1EKGJiPiT9IuTQ9nCG2PslkqR+KUMiYoS9MqTsAj9HhuTMkFhcYFyufxFmt/S4rIqVwmP8lY4GwwJwOnZwNLj/I2HwC+"
+        "pk=\20testuser@fedora.test.local"
+    )
```

Comment:
@pbrezina I wonder what would be the best way to store the ssh key in the new testing framework. Is it okay to have it as it is now? Or should it be moved to a file?

Reply:
I think we can move it to `data` directory, let me try that.

Reply:
We can but here we are not testing auth from key so doing all is like extra unrelated things, so prefer to use the pre-generated.

Reply:
Yes added in `data` directory, code is readable and clean.